src/sqlfluff/core/templaters/jinja.py

```diff
@@ -200,6 +204,152 @@ def __str__(self):  # pragma: no cover TODO?
         }
         return dbt_builtins
 
+    @staticmethod
+    def _generate_airflow_builtins():
+        """Generate the airflow builtins which are injected in the context.
+
+        https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html
+        """
+        # Copied from _generate_dbt_builtins
+        # This feels a bit wrong defining these here, they should probably
+        # be configurable somewhere sensible. But for now they're not.
+        # TODO: Come up with a better solution.
```

Comment:
Because this is a large amount of code, I would suggest creating a new module file, e.g. `src/sqlfluff/core/templaters/builtins/airflow.py` or similar.

The question about making it configurable is also a good one. Just thinking (not necessarily recommending), I wonder if we could structure this more like external code, i.e. the same directory/file structure used by `load_macros_from_path`. The more it resembles that structure, the easier it would be to make it configurable, either now or later.

Reply:
Yeah, putting it into a new module makes sense. I'll have a look at it.

Reply:
I tested around with loading the Airflow builtins exactly like an external library. There is a few problems with it still, i.e. I don't know yet, how I get this to work with Jinja filters, but in general I like this approach. WDYT?

Reply:
Sorry for the slow reply -- I'm not receiving GitHub email notifications recently for some reason.

I like it! One suggested change -- since the Airflow functionality is specific to the Jinja templater, I'm thinking we may want to reorganize things a little into this structure:
src/sqlfluff/core/templaters/jinja_builtins/
src/sqlfluff/core/templaters/jinja_builtins/airflow/

This emphasizes that it's for Jinja, and creates a place for future builtins, e.g. dbt if we wanted to organize them the same way.

Reply:
No worries. I'm in no particular rush with this one. I moved it around and fixed few more bugs related to loading it as a library.